tools/release/adot-operator-images-mirror/allowlist.yml

```diff
@@ -0,0 +1,7 @@
+# When adding a new repository to the allow list, make sure to have the full repository name (Host + Registry + Name)
+ghcr.io/open-telemetry/opentelemetry-operator/opentelemetry-operator:
```

Comment:
I wouldn't have a separate file for this.  Put it in https://github.com/aws-observability/aws-otel-collector/blob/main/tools/release/adot-operator-images-mirror/config.yaml directly in the source repos.

Reply:
Removed `allowlist.yml` and added to `config.yaml`, and also removed the code block you reference in the comment below + modified the `tagsInAllowlist` check to properly work as a result of the change.